CDCgov/prime-reportstream
# 12891 - Implement preview images to ReportStream website pages

This PR ...

Implement preview images to ReportStream website pages

Test Steps:
1. Log into RS
2. Check that each page has a title element and metadata for og:image and og:image:alt in the page source.

## Changes
![Screenshot 2024-02-26 at 5 10 47 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/18070643-f974-40ab-8981-762a3ac182f4)


## Checklist

### Testing


## Linked Issues
- Fixes #12891
